src/App.js

```diff
@@ -1,25 +1,34 @@
-import FindMember from './FindMember';
-import Person from './Person';
-import './App.css';
+import FindMember from "./components/findMember/FindMember";
```

Comment:
**Bonus**: Find way to import like this 
import FindMember from "./components/findMember"